cloud-deployment-guide/plugin-editor-version-compatibility.md

```diff
@@ -11,6 +11,11 @@ Specific versions of TinyMCE are compatible with premium plugins.
 Use the following matrix in conjunction with the optional parameters for specifying editor version and [plugin version]({{ site.baseurl }}/cloud-deployment-guide/editor-plugin-version) to ensure compatibility.
 
 > Important: Ensure the plugins are supported in your editor version when deploying premium plugins only[^footnote] via [Tiny Cloud]({{ site.baseurl }}/cloud-deployment-guide/features-only).
+> Also ensure you are using the latest script tag and substitute 'no-api-key' with your api key.
+```js
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/tinymce.min.js" referrerpolicy="origin></script>
```

Comment:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/tinymce.min.js" referrerpolicy="origin"></script>
```

Reply:
I put this together:  

http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/plugin-editor-version-compatibility/